File: adgweb3/scripts/deploy.js
async function main() {
    const [deployer] = await ethers.getSigners();
    console.log("Deploying contracts with the account:", deployer.address);
    console.log("Account balance:", (await deployer.getBalance()).toString());
    const Token = await ethers.getContractFactory("ADGToken");
    const token = await Token.deploy("Art de Grace", "ADG");
    console.log("Token address:", token.address);
  }
  main()
    .then(() => process.exit(0))
    .catch((error) => {
      console.error(error);
      process.exit(1);
    });

File: adgweb3/scripts/deploy.py
#!/usr/bin/3
from brownie import ADGToken
from connectors import get_account


def main(): 
    # Fetch the account 
    account = get_account()
    print(account)
    # Deploy contract 
    deploy_contract = ADGToken.deploy("Art de Grace", "ADG", {"from" : account}, publish_source=True) 
    # Print contract address 
    print(f"contract deployed at {deploy_contract}")
